### PR TITLE
Changes related to libgd

### PR DESCRIPTION
**Pull request by JPeterMugaas**
mingw-w64-libgd - 2.2.4
  - add fontconfig, libimagequant, libwebp, and xpm-nox as dependencies
  - fix gdlib.pc file
  - restore gdlib-config script.  The original CMake with patch does not build it and some old config secripts are still using it.
  - libimagequant is now in a new package so don't with CMake.  Use the installed one.
mingw-w64-libimagequant - 2.8.2
  - new package
  - add patch I made to prevent a preliminary check for compiler working.  That triggers a bug with /dev/nul in GCC (or is it a Windows "feature")
mingw-w64-libgphoto - 2.5.12
  - Update to latest version and remove unneeded parts of aptch.  Much is already in the source-code.

### CODE REVIEW

**Comment by Alexpux** [comment]
Fix my comments

**Comment by JPeterMugaas** [comment]
What comments are you referring to?

**Comment by Alexpux** [comment]
Look into commit. I'm add comment about download file name and format

**Comment by JPeterMugaas** [comment]
Forgive me for being dense but I can't any comments that you had made.

**Comment by Alexpux** [comment]
Look more carefully. I'm add comments to libimagequant PKGBUILD

**Comment by JPeterMugaas** [comment]
Oh.  Okay.  You were speaking about that someone@somehwere.com maintainer?  I have updated that with my name and removed some commented out stuff.

**Comment by Alexpux** [comment]
no. look more

File: mingw-w64-libimagequant/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+_realname=libimagequant
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=2.8.2
+pkgrel=1
+pkgdesc="Palette quantization library (mingw-w64)"
+arch=('any')
+url='https://github.com/ImageOptim/libimagequant'
+license=("GPL3" "MIT")
+source=("https://github.com/ImageOptim/${_realname}/archive/${pkgver}.tar.gz"
```
Comment (mati865) [inline mingw-w64-libimagequant/PKGBUILD]: It should look similar to [this](https://github.com/Alexpux/MINGW-packages/pull/2124/files#diff-dd3baa61dc27b6450a49707a7992afb8L24)

File: mingw-w64-libimagequant/PKGBUILD [hunk 2]
```diff
@@ -9,7 +9,7 @@ pkgdesc="Palette quantization library (mingw-w64)"
 arch=('any')
 url='https://github.com/ImageOptim/libimagequant'
 license=("GPL3" "MIT")
-source=("${pkgver}.tar.gz"::"https://github.com/ImageOptim/${_realname}/archive/${pkgver}.tar.gz"
+source=("https://github.com/ImageOptim/${_realname}/archive/${pkgver}.tar.gz"
```
Comment (mati865) [inline mingw-w64-libimagequant/PKGBUILD]: @Alexpux you are checking outdated diff

**Comment by JPeterMugaas** [comment]
If it helps, I did make an attempt at: https://github.com/Alexpux/MINGW-packages/pull/2136 to try to fix some of this.  I had closed the branch here.  This includes a minor fix for something that I had seen with rebuilding libimagequant as well as removing the shared dll stuff since it doesn't seem to work as well as I had originally thought.  HTH.

**Pull request merged**